README.md

```diff
@@ -1,24 +1,92 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# Blog-App <a name="about-project"></a>
 
-* Database creation
+**Blog-App** is a project developed using Ruby on Rails where you can add users, comments, posts and likes.
 
-* Database initialization
 
-* How to run the test suite
+## 🛠 Built With <a name="built-with"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Tech Stack <a name="tech-stack"></a>
 
-* Deployment instructions
+> Developed using Ruby on Rails.
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+    <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
 
-* ...
+### Key Features <a name="key-features"></a>
+- **Database that contains information about users, likes and posts**
+## 💻 Getting Started <a name="getting-started"></a>
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- You need to have ruby on rails installed in your computer.
+
+### Setup
+```
+- To clone the repository run this command - "https://github.com/Gleennkar/My_Blog.git"
+- cd blog-app
+```
+## 👥 Authors <a name="authors"></a>
+
+👤 **Odette Gleen**
+
+- GitHub: [@Gleennkar](https://github.com/Gleennkar)
+- Twitter: [@gleeny24](https://twitter.com/twitterhandle)
+- LinkedIn: [@gleeny](https://www.linkedin.com/in/gleeny-nkar-aa3917182)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
```

Comment:
- [ ] Kindly in the `Getting started` section add the instruction on how to install gems `bundle i`, setup database `rails db:setup` and run rails server `rails s`.

- [ ] Please add the future features that you will implement in the upcoming project milestones in the `Future Features` section to make your readme file more professional.